### PR TITLE
Create Literals

### PR DESCRIPTION
**Pull request by theAYgithub**
The literals.
costume2
Screenshot 2024-02-26 191148

### CODE REVIEW

**Comment by theAYgithub** [comment]
You can use custom slots when needed.

**Comment by mariocraft987** [comment]
👍

**Comment by theAYgithub** [comment]
Go to [this website](https://studio.penguinmod.com/#4220069985175) to get the code

**Comment by mariocraft987** [comment]
I see

**Comment by theAYgithub** [comment]
Click on the sprite on the top left to view code.

**Comment by mariocraft987** [comment]
I also made my [own extension](https://studio.penguinmod.com/#5685234766835l)

**Comment by RedMan13** [comment]
i find it kinda funny how we're just ignoring the block load error :trollface:

**Comment by mariocraft987** [comment]
:trollface:

File: static/extensions/theAYscratch/Literals [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    };
+
+    blocks.push({
+        opcode: `matrix`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `01 [MATRIX] 10`,
+        arguments: {
+            "MATRIX": {
+                type: Scratch.ArgumentType.MATRIX,
+                defaultValue: 1.00110101001111e+22,
```
Comment (RedMan13) [inline static/extensions/theAYscratch/Literals:296]: use `'000011110101100101010110'` as the matrix field can't handle a real number and requires a string containing the binary sequence

**Comment by lego7set** [comment]
what is the null block supposed to do? it doesn't actually return null?

**Comment by SharkPool-SP** [comment]
I dont see any use for this + A lot of the blocks already exist in other extensions

**Comment by TheShovel** [comment]
As mentioned by other people, there are errors and this doesn't have a use since there are a couple other extensions, including built-in ones that already do this, so i'm gonna close this

**Pull request closed without merging**